index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400;500;700&display=swap" rel="stylesheet">
+</head>
+<body class="main__container">
+    <!-- Creating the navigation menu bar -->
+  <header class="container_menu">
+    <nav class="container--nav">
+        <ul class="navegation--list">
+            <li class="navigation--option"><a>Portfolio</a></li>
+            <li class="navigation--option"><a>About</a></li>
+            <li class="navigation--option"><a>Contact</a></li>
+        </ul>
+        <div class="menu--image">
+            <img src="images\menuUnion.png" alt="Menu button"/>
+        </div>
+    </nav>
+  </header>
+  <!-- This section is the main part, containing heading and a brief description of my work-->
+  <main class="main__containerinfo">
+   <h1 id="titlePortfolio">Hey there. I’m Reynaldo. </h1>
+   <h2 id="developerPortfolio">I’m a software developer</h2>
+   <p id="descriptionPortfolio"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hesitate to contact me.</p>
+  </main>
+
+  <section class="container__icons">
+    <div class="brand--icon">
+      <a href="https://twitter.com/"><img src="images\brand-twitter-filled.png" alt="twitter logo" class="icon--image" /></a>
+    </div>
+    <div class="brand--icon">
+      <a href="https://google.com"><img src="images\brand-gmail.png" alt="gmail logo" class="icon--image"></a>
+    </div>
+    <div class="brand--icon">
+      <a href="https://www.linkedin.com/in/reynaldo-casta%C3%B1o/?locale=en_US"><img src="images\brand-linkedin.png"alt="linkedin logo" class="icon--image"></a>
+    </div>
+    <div class="brand--icon">
+        <a href="https://github.com/ReyAntonio12"><img src="images\brand-github-filled.png" alt="github logo" class="icon--image"></a>
+    </div>
+    <div class="brand--icon">
+        <a><img src="images\angellist.png" alt="angellist logo" class="icon--image"></a>
+    </div>
+  </section>
```

Comment:
- [ ] It is important to make use of semantic tags as much as possible in our application and as such i will suggest rather than using the non-semantic `div` tag as container for the social media icons, you make use of the semantic `ul` tag and each link would be wrapped in an `li` tag as a list item. see the example below for more reference :+1: 
```html
<ul class="socials">
  <li>
    <a href="https://github.com/"><img src="./assets/github.png" alt="github"/></a>
  </li>
  <li>
    <a href="https://linkedin.com/in/"><img src="./assets/linkedin.png" alt="linkedin"/></a>
  </li>
</ul>
```